scripts/interact-goerli.js

```diff
@@ -0,0 +1,44 @@
+const ALCHEMY_GOERLI_API_KEY = process.env.ALCHEMY_GOERLI_API_KEY;
+const CRYPTO_WALLET_PRIVATE_KEY = process.env.CRYPTO_WALLET_PRIVATE_KEY;
+const GOERLI_CONTRACT_ADDRESS = process.env.GOERLI_CONTRACT_ADDRESS;
```

Comment:
Hey Mari, everything is looking pretty good. The only change I'd make here is in the assignment of GOERLI_CONTRACT_ADDRESS to NEXT_PUBLIC_GOERLI_CONTRACT_ADDRESS instead of GOERLI_CONTRACT_ADDRESS.